cronjob.js

```diff
@@ -1,45 +1,68 @@
 const https = require('https');
 const moment = require('moment');
+const nodemailer = require("nodemailer");
```

Comment:
no longer required.

We can also remove it from npm

Reply:
ok